src/main/java/org/bladerunnerjs/contrib/contentplugin/appcache/AppcacheManifestBuilder.java

```diff
@@ -52,16 +45,11 @@ public AppcacheManifestBuilder(BRJS brjs, BundleSet bundleSet, String brjsVersio
 	 */
 	public String getManifest(RequestMode requestMode) throws PropertiesException, ContentProcessingException, ConfigException, MalformedTokenException
 	{
-		String manifest = getManifestHeader();
-        if(version != null)
```

Comment:
I'm not sure this 'if' guard should have been removed; the need for it probably wasn't clear (or I've misunderstood something about the refactor).
Without it, there are annoying delays in dev when you disable appcache before you see the uncached files. This is due to the way appcache works:

* ./brjs serve -v 1.2.3
* Load page, and get served <html manifest="dev.appcache" />
* dev.appcache contents:
```
    # v 1.2.3
    files...
```
* Appcache caches all the files...

* Stop brjs
* ./brjs serve
* Reload page, and get served <html manifest="dev.appcache" /> from appcache
* dev.appcache contents:
```
    # v 
    files...
```
* Appcache tries to recache all the files, even though we're disabling appcache ...
* This takes time. If you refresh the page before all the files have recached, you still get the cached version.

With the 'if' guard, when you reload the page dev.appcache just contains '#v' and no files. This causes the appcache to refresh instantly without trying to recache all the files, and you see the uncached files as soon as you reload.